pkg/k8s/utils.go

```diff
@@ -81,3 +85,95 @@ func GenerateConfigMapFromDir(dir string, name string, namespace string, skipFil
 
 	return configMap, nil
 }
+
+type SecretVolume struct {
+	Name      string
+	SecretName string
+	Key string
+	SubPath string
+	MountPath string
+
+
+}
+
+func ReadEnvFile(filePath string) ([]corev1.EnvVar, []SecretVolume, error) {
```

Comment:
Low-value suggestion, but I would suggest naming the function something else in order to avoid thinking this injects env vars from a file (similar to [dotenv](https://github.com/joho/godotenv)), which was my initial reaction to seeing the function name.

```suggestion
func ParseEnvFile(filePath string) ([]corev1.EnvVar, []SecretVolume, error) {
```

Reply:
But it injects env vars from a file?

Reply:
Never mind this. Let's talk semantics another time 😂